### ISSUE TITLE
organisation in byOrganisation property has type person

### ISSUE THREAD

**Issue by wmelder** [beeldengeluid/beng-lod-server#357]
Example showing data with wrong type:
```sh
curl -v -L -H "Accept: text/turtle" http://data.beeldengeluid.nl/id/program/2101712250239865931
```

**Comment by wmelder** [comment]
Voor deze resource: http://data.beeldengeluid.nl/id/program/2101712250239865931
```json
    {
      "@id": "_:N38abb6acb0fc4012bbe4db139ad148d8",
      "@type": "sdo:PerformanceRole",
      "sdo:byArtist": {
        "@id": "gtaa:61423"
      }
    },
    {
      "@id": "gtaa:61423",
      "@type": "sdo:Person",
      "sdo:additionalType": {
        "@id": "skos:Concept"
      },
      "skos:prefLabel": {
        "@language": "nl",
        "@value": "Metropole Orkest"
      }
    }
```

**Comment by wmelder** [comment]
According to the definition of [byArtist ](https://schema.org/byArtist) it is either a MusicGroup or A Person:
[MusicGroup](https://schema.org/MusicGroup)
[Person](https://schema.org/Person)
- [ ] check where this is handled in the code.

**Comment by wmelder** [comment]
daan-mapping-schema-org.ttl: 
```turtle
###  https://schema.org/byArtist
<https://schema.org/byArtist> <http://data.rdlabs.beeldengeluid.nl/schema/hasDaanPath> "nisv.performer, performer.name, resolved_value".

```

The code is suggests that we can distinguish Person and Organisation:
```python
                    # type is set to Person or Organization depending on the property range
                    # additionalType is set to SKOS concept
                    self.graph.add(
                        (
                            skos_concept_node,
                            RDF.type,
                            URIRef(property_description["range"]),
                        )
                    )
```

DAAN field: 

![Image](https://user-images.githubusercontent.com/1220375/232006034-f900d22c-8698-4347-ad87-ea9d8e465526.png)

@mwigham Is it correct, that we cannot distinguish between Person and MusicGroup, because the DAAN field is not making a difference?

**Comment by mwigham** [comment]
That is correct. The value of the performer field in our metadata can be a person or a musical group. 
Potentially we could check the schema of the GTAA concept, that would tell us which one is correct (although I think that the division between Persoonsnamen and Namen is also not always perfect)

**Comment by wmelder** [comment]
>That is correct. The value of the performer field in our metadata can be a person or a musical group.

So, by default it is always a Person in the property-description? 

>Potentially we could check the schema of the GTAA concept, that would tell us which one is correct (although I think that the division between Persoonsnamen and Namen is also not always perfect)

I am not sure if we want to incorporate another dependency in the payload_to_rdf, but indeed we could read the ConceptScheme from the triple store or by dereferencing the URI. Indeed, it is not always correct, like 'Diverse uitvoerenden' being a person name: http://data.beeldengeluid.nl/gtaa/1233738

**Comment by mwigham** [comment]
Yes, at present it is set to be a Person. We could change it to be sdo:Thing, as for sdo:mentions, which can also include both persons and organisations.
